qemu/tests/fio_windows.py

```diff
@@ -23,8 +21,10 @@ def run(test, params, env):
     """
 
     install_path = params["install_path"].rstrip("\\")
-    log_file = params.get("log_file")
-    fio_cmd = params.get("fio_cmd")
+    log_file = params.objects("log_file")
+    file_name = params.objects("file_name")
```

Comment:
Suggest adding prefix `fio_` for these two params.

Reply:
Updated, thanks~